### PR TITLE
simple styling for createForm

### PR DESCRIPTION
**Pull request by mun625**
stylileelele

### CODE REVIEW

File: shared/components/CreateForm.js [hunk 1]
```diff
@@ -115,13 +141,12 @@ class CreateForm extends Component {
...
 
     const value = this.refs.form.getValue();
-    const location = this.props.location;
 
     const img = this.state.img;
     const data = JSON.stringify({
           title: String(value.title),
-          lat: location.latitude,
-          lng: location.longitude,
+          lat: 30,
```
Comment (ochanje210) [inline shared/components/CreateForm.js]: why did you change it to a magic number?

Comment (ochanje210) [inline shared/components/CreateForm.js]: @mun625

Comment (mun625) [inline shared/components/CreateForm.js]: oh no

**Comment by ochanje210** [comment]
LGTM

**Comment by elegant651** [comment]
LGTM
